.github/workflows/test-job.yaml

```diff
@@ -134,45 +135,31 @@ jobs:
       run: make test
       working-directory: ${{env.working-directory}}
 
-    - name: Install Kind
-      run: |
-        curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.11.1/kind-linux-amd64
-        chmod +x ./kind
-        sudo mv ./kind /usr/local/bin/kind
-
-    - name: Checkout Python
-      uses: actions/checkout@v2
-
-    - name: Setup Python
-      uses: actions/setup-python@v2
-      with:
-        python-version: '3.x'
-
-    - name: Install Python dependencies
-      run: pip install -r ./tests/config/requirements.txt
-
     - name: Set up Docker
       uses: docker-practice/actions-setup-docker@master
 
     - name: Build Docker Image - Operator
-      run: IMG=kuberay/operator:${{ steps.vars.outputs.sha_short }} make docker-image
+      run: |
+        IMG=kuberay/operator:${{ steps.vars.outputs.sha_short }} make docker-image
+        docker save -o /tmp/operator.tar kuberay/operator:${{ steps.vars.outputs.sha_short }}
       working-directory: ${{env.working-directory}}
 
-    - name: Build Docker Image - Apiserver
-      run: docker build -t kuberay/apiserver:${{ steps.vars.outputs.sha_short }} -f apiserver/Dockerfile .
+    - name: Upload Artifact Operator
+      uses: actions/upload-artifact@v2
```

Comment:
I am new to this plugin,  How does upload-artifact work?

Reply:
It will upload a file that we can use later in other jobs. I want to reuse the images built here in other following jobs.